src/Database/LSMTree/Internal/RunAcc.hs

```diff
@@ -211,7 +211,6 @@ addLargeKeyOp ::
 addLargeKeyOp racc@RunAcc{..} k e =
   assert (not (PageAcc.entryWouldFitInPage k e)) $ do
     modifyPrimVar entryCount (+1)
-    Bloom.insert mbloom k
```

Comment:
Oh, and this one too (see https://github.com/IntersectMBO/lsm-tree/pull/723/files#r2090655471)

Reply:
Right.

Reply:
Done.